delphi-checks/src/main/resources/org/sonar/l10n/delphi/rules/community-delphi/RedundantInherited.html

```diff
@@ -0,0 +1,25 @@
+<h2>Why is this an issue?</h2>
+<p>
+  It is possible to use a bare <code>inherited</code> in any method implementation,
+  even if there is no base method to call.
+  Although this statement has no behavioral effect, it is misleading and indicates a possible programming mistake.
+</p>
+<p>
+  Since the <code>inherited</code> keyword works regardless of whether the method is marked as <code>override</code>,
+  it can also lead to unexpected behavioral change if corresponding ancestor methods are added later.
+</p>
```

Comment:
Let's reflow these `<p>` blocks and get the line lengths under 100.
```suggestion
<p>
  It is possible to use a bare <code>inherited</code> in any method implementation, even if there is
  no base method to call. Although this statement has no behavioral effect, it is misleading and
  indicates a possible programming mistake.
</p>
<p>
  Since the <code>inherited</code> keyword works regardless of whether the method is marked as
  <code>override</code>, it can also lead to unexpected behavioral change if corresponding ancestor
  methods are added later.
</p>
```